worker/src/Channel/Request.cpp

```diff
@@ -121,13 +136,15 @@ namespace Channel
 		jsonResponse["id"]       = this->id;
 		jsonResponse["accepted"] = true;
 
-		this->channel->Send(jsonResponse);
+		Request::channel->Send(jsonResponse);
 	}
 
 	void Request::Accept(json& data)
 	{
 		MS_TRACE();
 
+		MS_ASSERT(Request::channel != nullptr, "channel unset");
+
```

Comment:
Let's replace it with `MS_ASSERT(Request::channel, "channel unset");` and let's remove the empty line between both `MS_ASSERT()`s.